website/src/pages/showcase.js

```diff
@@ -14,13 +14,15 @@ import ThemedImage from '@theme/ThemedImage';
 import {Section} from './index';
 import IconExternalLink from '../theme/Icon/ExternalLink';
 
-const renderApp = (app, i) => {
-  const imgSource = !app.icon.startsWith('http')
-    ? useBaseUrl('img/showcase/' + app.icon)
-    : app.icon;
+const renderApp = (app, i) => <AppBox app={app} key={`app-${app.name}-${i}`} />;
+
+const AppBox = ({app}) => {
+  const imgSource = useBaseUrl(
```

Comment:
This was necessary to keep the # of hook calls consistent.  [`useBaseUrl` handles FQDNs](https://github.com/facebook/docusaurus/blob/2f75979bc5b4fd971225098e1c122efdd71d6e6f/packages/docusaurus/src/client/exports/__tests__/useBaseUrl.test.tsx#L41).